scripts/verify-brevity.py

```
@@ -0,0 +1,338 @@
+                   r"|\b\d{1,2}:\d{2}\s*(?:[ap]\.?m\.?)?(?:\s+[A-Z][a-zA-Z]+)?"
+                   r"|\b\d{1,2}\s*[ap]\.?m\.?(?:\s+[A-Z][a-zA-Z]+)?"),
+    # The token carries whatever changes the claim: a currency symbol, a
+    # sign, a percent, or a unit. `$4` and `EUR4` are different prices, `-3.8%`
+    # and `+3.8%` opposite directions, `9px` and `8px` different grids.
+    # Separators are only real between digits -- anchoring the tail on a digit
+    # keeps sentence punctuation out, so retightening "took 22, which" into
+    # "took 22. It" is not read as a changed value.
+    # A slash denominator is part of the rate: `$4/M` and `$4/B` differ by a
+    # factor of a thousand while every other token compares equal.
+    ("numerals", r"(?<![A-Za-z0-9_.])[$\u20ac\u00a3]?[+-]?\d(?:[\d,.]*\d)?"
+                 r"(?:%|[A-Za-z]{1,3}\b)?(?:/[A-Za-z]{1,6}\b)?"),
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)